docs-source/content/userguide/managing-fmw-domains/soa-suite/Logging-Exporter-Setup.md

```diff
@@ -0,0 +1,126 @@
+# Weblogic Logging Exporter Setup to publish logs into Elasticsearch
```

Comment:
Saw that Mark commented on this several times, but it's important to consistently use official product names with correct spelling including capital letters.  Weblogic -> WebLogic